### PR TITLE
Add UK Payment Initiation docs for all supported languages

### PR DESCRIPTION
**Pull request by skansal-plaid**

### CODE REVIEW

File: go/server.go [hunk 1]
```diff
@@ -206,6 +208,7 @@ func transactions(c *gin.Context) {
 	})
 }
 
+// This functionality is only relevant for the UK Payment Initiation product.
 // Retrieve Payment for a specified Payment ID
 // https://plaid.com/docs/#payment-initiation
```
Comment (lkosak) [inline go/server.go]: Can we remove?

File: ruby/app.rb [hunk 2]
```diff
@@ -225,6 +226,7 @@
     institution: institution_response['institution'] }.to_json
 end
 
+# This functionality is only relevant for the UK Payment Initiation product.
 # Retrieve Payment for a specified Payment ID
 # https://plaid.com/docs/#payment-initiation
```
Comment (lkosak) [inline ruby/app.rb]: Same here—remove?

File: go/server.go (hunk 1 above)
Comment (skansal-plaid) [reply go/server.go]: https://github.com/plaid/quickstart/pull/94/commits/e5a3cdfd703f17d525a395e01585a027aa8acf5a

File: ruby/app.rb (hunk 2 above)
Comment (skansal-plaid) [reply ruby/app.rb]: https://github.com/plaid/quickstart/pull/94/commits/e5a3cdfd703f17d525a395e01585a027aa8acf5a

**Pull request merged**